### ISSUE TITLE
Missing createNpc 

### ISSUE THREAD

**Issue by bostrt** [shawncplus/soma#4]
Non-existent `createNpc()` function is called when clicking `+` button to add NPC. Can this be added? Thanks :smile: 

References:
https://github.com/shawncplus/soma/blob/master/src/soma-app/soma-app.html#L262
https://github.com/shawncplus/soma/blob/master/src/soma-app/ranvier-service.html#L22-L53

**Comment by shawncplus** [comment]
Yeah I have a lot of work to do on Soma, once the holidays are over I have a huge list of things for Ranvier/Neuro/Soma
